editor/project_manager.cpp

```diff
@@ -659,39 +686,10 @@ class ProjectDialog : public ConfirmationDialog {
 		}
 	}
 
-	void _remove_created_folder() {
-		if (!created_folder_path.is_empty()) {
-			Ref<DirAccess> d = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
-			d->remove(created_folder_path);
-
-			create_dir->set_disabled(false);
-			created_folder_path = "";
-		}
-	}
-
 	void cancel_pressed() override {
```

Comment:
This function doesn't seem to be called anywhere 🤔
Also it's probably redundant, because simply closing the dialog seems to achieve the same.

Reply:
It's an override and it does get called, but I removed it anyways